docs/apis-tools/web-modeler-api/index.md

```diff
@@ -16,15 +16,15 @@ Web Modeler provides a REST API at `/api/*`. Clients can access this API by pass
 
 ## OpenAPI documentation
 
-A detailed API description is available as [OpenAPI](https://www.openapis.org/) specification at [https://modeler.cloud.camunda.io/swagger-ui/index.html](https://modeler.cloud.camunda.io/swagger-ui/index.html).
+A detailed API description is available as [OpenAPI](https://www.openapis.org/) specification at [https://modeler.cloud.camunda.io/swagger-ui/index.html](https://modeler.cloud.camunda.io/swagger-ui/index.html)
+for SaaS and at [http://localhost:8070/swagger-ui.html](http://localhost:8070/swagger-ui.html) for Self-Managed
+installations.
 
 ## Authentication
 
 To authenticate for the API, generate a JWT token and pass it in each request; guidance on this is provided in the following sections.
 
-### Obtain JWT token
-
-**Example:**
+### Authentication in the cloud
```

Comment:
Hmm, I just noticed our API pages are inconsistent in how we refer to C8 SaaS. No immediate action required, but I'll add this to the backlog.

Reply:
https://github.com/camunda/camunda-platform-docs/issues/2098